decaf377-fmd/src/clue_key.rs

```diff
@@ -61,6 +84,7 @@ impl ExpandedClueKey {
     /// # Errors
     ///
     /// `precision_bits` must be smaller than [`MAX_PRECISION`].
+    /// `precision_bits` must be smaller or equal to the `ExpandedClueKey` precision.
```

Comment:
I don't think this condition should cause an error -- we should expand the clue key as needed, as long as it's less than `MAX_PRECISION`, and in fact the code works like that, so we don't need any extra conditions.